bioframe/core/arrops.py

```diff
@@ -264,6 +264,22 @@ def _overlap_intervals_legacy(starts1, ends1, starts2, ends2, closed=False, sort
 
     return overlap_ids
 
+def get_pseudo_segment(starts, ends):
```

Comment:
maybe a more instructive name would be `_convert_points_to_len1_segments`? cc @nvictus

Reply:
I would probably agree with Geoff here. "pseudo-segment" sound like a decent option to me, but Geoff's suggestion will make it more readable. Also, maybe it'd worth adding the intended usecase/problem that is being solved to the docstring?

Reply:
I will add few use cases in docstring and update the change the function namae.